ts/src/pro/huobi.ts

```diff
@@ -630,18 +609,6 @@ export default class huobi extends huobiRest {
         }
     }
 
-    handleOrderBookSubscription (client: Client, message, subscription) {
-        const symbol = this.safeString (subscription, 'symbol');
-        const limit = this.safeInteger (subscription, 'limit');
-        if (symbol in this.orderbooks) {
-            delete this.orderbooks[symbol];
-        }
-        this.orderbooks[symbol] = this.orderBook ({}, limit);
-        if (this.markets[symbol]['spot'] === true) {
```

Comment:
btw, this check was no longer necessary, as `handleOrderBookSubscription` method itself is not hooked when market is not `spot`: https://github.com/ccxt/ccxt/blob/490e83521834a451e59ae91b2a8de84b8c7c2458/ts/src/pro/huobi.ts#L352